<?xml version="1.0" encoding="UTF-8"?>
<launch>
  <arg name="follow" default="no"/>

  <param name="mdelay" type="int" value="75" />
  <param name="duty_cycle" type="int" value="50" />
  <param name="target" type="int" value="0" />
  <param name="cc" type="bool" value="True" />
  <param name="ooi" type="string" value="missing" />

  <group if="$(eval arg('follow') == 'no')">
    <node name="client" pkg="sphere_control" type="joystick_control_client.py" output="screen"/>
  </group>
  <group if="$(eval arg('follow') == 'yes')">
    <node name="find_oois" pkg="sphere_control" type="find_oois.py" output="screen"/>
    <node name="home_in" pkg="sphere_control" type="home_in.py" output="screen"/>
  </group>
  <!--node name="gui" pkg="sphere_control" type="sphere_gui.py" output="screen"/-->
</launch>